roles/tackle/defaults/main.yml

```diff
@@ -24,17 +24,17 @@ hub_deployment_name: "{{ hub_service_name }}"
 hub_deployment_replicas: "1"
 hub_deployment_strategy: "Recreate"
 hub_container_name: "{{ hub_service_name }}"
-hub_container_limits_cpu: "500m"
-hub_container_limits_memory: "800Mi"
-hub_container_requests_cpu: "100m"
-hub_container_requests_memory: "350Mi"
+hub_container_limits_cpu: "1"
```

Comment:
@djzager do we really need limits?

Reply:
Seems like we should have a limit to be a good citizen on the cluster.  That said, I don't know if 1 core is appropriate for out-of-the-box but felt sure that 100m was not enough.